pkg/management/component.go

```diff
@@ -144,6 +144,9 @@ type Component interface {
 	DeleteGroup(ctx context.Context, realmName string, groupID string) error
 	GetAuthorizations(ctx context.Context, realmName string, groupID string) (api.AuthorizationsRepresentation, error)
 	UpdateAuthorizations(ctx context.Context, realmName string, groupID string, group api.AuthorizationsRepresentation) error
+	PutAuthorization(ctx context.Context, realmName string, groupID string, group api.AuthorizationsRepresentation) error
```

Comment:
Rename to AddAuthorizations (name the feature, not the way it is called through REST)

Reply:
Corrected in commit e65d1abb21021efc38ddf4adf3ef93c6273bd283